buildSrc/src/main/groovy/net/minecraftforge/forge/tasks/checks/CheckATs.groovy

```diff
@@ -171,7 +171,7 @@ abstract class CheckATs extends CheckTask {
     }
 
     private static List<String> joinBack(TreeMap<String, ATParser.Entry> lines, Map<String, InheritanceData> inheritance, IMappingFile mappings) {
-        final data = []
+        final data = [] as List<String>
```

Comment:
```suggestion
        final List<String> data = []
```

Reply:
thats how was done in 1.20.x branh, but fair enough